docs/guides/end-to-end-testing/migration/selenium-to-cypress.mdx

```diff
@@ -0,0 +1,431 @@
+---
+title: Migrating from Selenium to Cypress
+slug: /guides/end-to-end-testing/selenium-to-cypress
+---
+
+:::info
+
+## <Icon name="graduation-cap" /> What you'll learn
+
+- Why to migrate from Selenium to Cypress
+- How to migrate Selenium tests to Cypress
+
+:::
+
+## Introduction
+
+While both Selenium and Cypress are popular tools for web automation,
+each framework possesses distinct strengths and weaknesses. For instance,
+Cypress stands out as one of the easiest automation frameworks to begin with,
+while Selenium boasts the status of being one of the oldest and most widely used frameworks.
+
+:::caution
+
+If you see any inaccuracies with this guide or feel like something has been
+misrepresented, please
+[start a discussion here](https://github.com/cypress-io/cypress/discussions/new?category=general).
+
+:::
+
+Taking a closer look at Selenium WebDriver reveals its longevity and extensive library of
+documentation and troubleshooting guides. Supported by an active community, Selenium benefits
+from robust documentation across multiple languages including Java, JavaScript, Python, C#, and
+Ruby. Its widespread adoption is further evidenced by its integration into various low-code
+tools, underscoring its versatility in browser interaction.
+
+A key differentiator of Cypress, not only from Selenium but also from other test automation frameworks,
+is its unique approach of running tests inside an interactive browser rather than against one.
+This approach minimizes flakiness and grants users greater control over both the framework and the
+application under test. Cypress's ability to wait for page loading without explicit declaration allows
+for patient execution until the application is fully ready. Additionally, Cypress simplifies test recording
+without requiring complex configurations or setups, facilitating easy troubleshooting and bug
+reporting with access to screenshots and videos.
+
+Furthermore, the recent addition of component testing to Cypress has fostered collaboration between testing
```

Comment:
```suggestion
Furthermore, the addition of component testing to Cypress has fostered collaboration between testing
```

Component testing was officially released as GA (Generally Available) in [cypress@11.0.0](https://docs.cypress.io/guides/references/changelog#11-0-0) in Aug 2022 for React, Next.js, Angular, and Vue. I wouldn't use "recent" for an event which happened more than one year ago.